app/Modules/Group/groups.php

```diff
@@ -151,4 +171,7 @@
     'role_permissions' => [
         'coordinator' => [1001,1002,1003,1004,1005,1006, 1007, 1008]
     ],
+    'role_permissions' => [
```

Comment:
This looks wrong to me. Having a second "role_permissions" key in this object will result in the first's value being overwritten.

Reply:
good catch.  I tested the coordinator isContact functionality but didn't notice roles needed weren't granted.